src/overlays/actors/ovl_En_Bombf/z_en_bombf.h

```diff
@@ -6,9 +6,20 @@
 
 struct EnBombf;
 
+typedef void (*EnBombfActionFunc)(struct EnBombf*, struct GlobalContext*);
+
 typedef struct EnBombf {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0xC4];
+    /* 0x014C */ ColliderCylinder bombCollider;
+    /* 0x0198 */ ColliderJntSph jntSphList;
+    /* 0x01B8 */ ColliderJntSphItem explosionCollider[1];
+    /* 0x01F8 */ s16 timer;
+    /* 0x01FC */ EnBombfActionFunc actionFunc;
+    /* 0x0200 */ s32 unk_200; // this var always seems to be true, maybe gets set by another actor?
```

Comment:
I could be mistaken but this seems to be set to true on init so that's probably why it seems to always be that way?
```suggestion
    /* 0x0200 */ s32 unk_200;
```